INSTALL.md

```diff
@@ -105,6 +105,81 @@ For Example:
 ./tools/tpm2_getrandom 4
 ```
 
+### Build example
+
+#### Fedora 27
```

Comment:
Just say Fedora, otherwise this doc will be outdated soon (Fedora 28 is going to be released in a of weeks).